core/src/main/kotlin/org/evomaster/core/problem/externalservice/httpws/param/HttpWsResponseParam.kt

```diff
@@ -73,7 +73,7 @@ class HttpWsResponseParam(
     fun getResponseBodyBasedOnStatus(): String {
         val statusCode = getHttpStatusCode()
 
-        if (arrayOf(204, 304).contains(statusCode)) {
+        if (arrayOf(100..199, 204, 304).contains(statusCode)) {
```

Comment:
this is very inefficient.
each time this method is called, you create on the heap an array of 102 elements, which is then thrown away. either put it in a static variable, or better change the condition, eg, something like `if( (c>=100 && c<=199) || c==204 || c==304)`, where `c` is the status code